app/src/main/java/com/example/scoutconcordia/MapsActivity.java

```diff
@@ -282,6 +269,64 @@ else if (reason == GoogleMap.OnCameraMoveStartedListener.REASON_API_ANIMATION) {
         }
     }
 
+    // This is the method that will be used to encrypt all of the input files during the app startup.
+    public void encryptAllInputFiles()
```

Comment:
Doesn't really encrypt all files, should obtain a list of files to encrypt / decrypt (the decrypt functionality should be included) and then work from there

Reply:
Fixed this issue by creating a file called "filestoencrypt" in the raw folder. This file contains a list of filenames of files that we would like to encrypt. We can add more file names to this as we progress further. The data files should be place within the res/raw folder.